index.html

```diff
@@ -85,5 +85,29 @@
     of code. If it needs to be a bit bigger, it will be.
   </li> 
 </ol> 
+
+<p>
+In addition to these stated goals, we try our best to uphold the following
+values when considering contributions to the project:
+</p>
+
+<ul>
+  <li>
+    Never break configuration files or existing workflows. Breaking changes
+    require a major version bump (v4 → v5).
```

Comment:
Agreed. The “immediately release a new major version” bit makes sense to me, we should consider it when running into a similar case in the future.